library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/AdministratorDoctrineRepository.php

```diff
@@ -49,4 +49,16 @@ public function getInnerGlobalAdmin()
 
         return $privateAdmin;
     }
+
+    /**
+     * @param string $username
+     * @return null| AdministratorInterface
+     */
+    public function findPlatformAdminByUsername(string $username)
+    {
+        return $this->findOneBy([
+            'username' => $username,
+            'brand' => null
```

Comment:
Not necessary, but I will add _company => null_ here to be ultra-safe.

Reply:
You got it :)